frontend/src/api/types.ts

```diff
@@ -94,9 +94,9 @@ export type CompoundDto = {
 
 export type ExperimentDto = Dto<{
   project: number;
-  compound: number;
-  target: number;
-  methodology?: number;
+  compound: [number, string];
```

Comment:
I am not sure if we need it, but I decided to keep it here because it's very small and may be useful in future. This way the response is more self-contained.